float32 stamp
string[] node_id
string[] robot_id
